modules/core/src/core/animation-loop.js

```diff
@@ -205,20 +225,11 @@ export default class AnimationLoop {
       return;
     }
 
-    this._setupFrame();
-    this._updateCallbackData();
-
-    // call callback
-    this.onRender(this.animationProps);
-    // end callback
-
-    // clear needsRedraw flag
-    this._clearNeedsRedraw();
+    const promise = this.redraw();
 
-    if (this.offScreen && this.gl.commit) {
-      // https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/commit
-      // commit returns a Promise
```

Comment:
According to the linked doc, `commit` is a void function. Maybe this has changed as the offscreen feature evolved?

Reply:
Good catch. I haven't checked back on the doc for a while.